lib/identity/hostdata/s3.rb

```diff
@@ -16,22 +16,6 @@ def initialize(bucket:, env:, region:, s3_client: nil, logger: nil)
         @s3_client = s3_client
       end
 
-      def download_configs(configs)
-        configs.each do |s3_path, local_path|
-          download_config(s3_path, local_path)
-        end
-      end
-
-      private
-
-      def s3_client
-        @s3_client ||= Aws::S3::Client.new(
-          region: region,
-          http_open_timeout: 5,
-          http_read_timeout: 5
-        )
-      end
-
       def download_config(s3_path, local_path)
```

Comment:
Two ideas:
1. WDYT if we made these keyword arguments to be clearer
2. What about renaming to `download_file` since these could be configs, secrets, etc etc?

```suggestion
      def download_file(s3_path:, local_path:)
```